README.md

```diff
@@ -77,6 +77,7 @@ $server = $forge->createServer([
     "provider"=> ServerProviders::DIGITAL_OCEAN,
     "credential_id"=> 1,
     "name"=> "test-via-api",
+    "type"=> ServerTypes::APP,
```

Comment:
Note that the default type is `app`.

Reply:
👍 I wanted to use an example in there, but didn't want it to be confusing for anyone reading it, so went with the default.

Reply:
Makes sense!